frontend/modules/recipe_form/components/DirectionList.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import {
+    injectIntl,
+    IntlProvider,
+    defineMessages,
+    formatMessage
+} from 'react-intl';
+import { TextArea } from './FormComponents'
+
+var Instruction = injectIntl(React.createClass({
+  getInitialState: function() {
+    return {
+      step: this.props.data ? this.props.data.step : 0,
+      title: this.props.data ? this.props.data.title : ''
+    };
+  },
+
+  update: function(name, value) {
+    this.setState({ [name]: value });
+    if(this.props.change) {
+      this.props.change(name, value);
+    }
+  },
+
+  render: function () {
+    const {formatMessage} = this.props.intl;
+    const messages = defineMessages({
+      step_label: {
+        id: 'instructions.step_label',
+        description: 'Step # label',
+        defaultMessage: 'Step {step, number}',
+      },
+      textarea_placeholder: {
+        id: 'instructions.textarea_placeholder',
+        description: 'Step # label',
+        defaultMessage: 'Instruction',
+      }
+    });
+
+    return (
+      <div className="instruction row">
+        <label className="col-xs-1">{ formatMessage(messages.step_label, {step: this.state.step}) }:</label>
+        <TextArea name="title" type="text" placeholder={ formatMessage(messages.textarea_placeholder) } size="col-xs-10" change={ this.update } value={ this.state.title }/>
+        <div className="col-xs-1">
+          <button onClick={ this.props.deleteInstruction }>
```

Comment:
You should add some color/style to this button. Something like: `className="btn btn-danger"`. You could even consider using the [glyph-icons](http://getbootstrap.com/components/#glyphicons-glyphs) that bootstrap provides.